schemas/delta.json

```diff
@@ -67,155 +67,6 @@
           }
         }
       }
-    },
-    "subscribe": {
```

Comment:
So is this completely gone? I thought from the description that you were moving things around. I do not see where this went.

Completely agree with the intent and direction of this PR though.

Reply:
(un)subscribe was is in master in delta.json and then at https://github.com/SignalK/specification/tree/c31e54f047d5e50c961d24b9898f13d5c165ed2e/schemas/messages